R/pmax-method.R

```diff
@@ -0,0 +1,22 @@
+#
+# Start Preamble to set up generic pmax
+#
+existingPmaxGeneric <- getGeneric("pmax")
+if (!is.null(existingPmaxGeneric)) {
+  if (!all(existingPmaxGeneric@signature == c("..."))) {
+    warning(paste0("Redefining pmax generic function with new signature \"...\", ",
+                   "while there already is a pmax generic with signature: ", existingPmaxGeneric@signature))
```

Comment:
```suggestion
    warning("Redefining pmax generic function with new signature \"...\", ",
            "while there already is a pmax generic with signature: ", existingPmaxGeneric@signature)
```
`warning` will already `paste0` it's args. However, what exactly can `existingPmaxGeneric@signature` be, a vector of strings? In that case you might need `paste0(existingPmaxGeneric@signature, collapse = ", ")` or so